src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        Colors c = Colors.values()[index];
```

Comment:
```suggestion
        Colors color = Colors.values()[index];
```
як метод буде рости c - стане не інформативним